src/character.h

```diff
@@ -241,8 +243,8 @@ class Character : public Creature, public visitable<Character>
         /** Toggles a trait on the player and in their mutation list */
         void toggle_trait(const std::string &flag);
         /** Add or removes a mutation on the player, but does not trigger mutation loss/gain effects. */
-        void set_mutation(const std::string &flag);
```

Comment:
I think it'l be clearer what's hapening here if you make a set_mutation()/set_mutation_nowarn() pair of methods where the _nowarn() varion does the actual work and the other just does the sanity check and calls _nowarn().